examples/notebooks/aim.ipynb

```diff
@@ -0,0 +1,198 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4df03fd2",
```

Comment:
Are those ids randomly generated? If yes then we should check if there is a way to make them deterministic. Otherwise we'll always get a lot of changes every time someone regenerates the notebooks even if the actual notebook content didn't change.

Reply:
Fixed it by adding a new pre-commit hook.

Reply:
Relying on pre-commit hook seems a bit dangerous to me as there is no guarantee that they will be run. Adding something like that in the `create_example_notebooks.py` script before saving the notebook might work:

```python
for i, cell in enumerate(nb.cells):
    cell.id = str(i)
```